src/neuroconv/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 from pynwb.file import NWBFile
 
+from ._dlc_utils import add_subject_to_nwbfile
```

Comment:
Unfortunately, this line here indirectly makes it exposed since a parent `__init__.py` somewhere along the chain will do a directed import from this public submodule

Meaning anyone working in a minimal installation, or an installation using extras that do not include DLC, will encounter import errors

Reply:
of, apparently this is not the crucial change as long as imports in `._dlc_utils` happen at the top of the file